go.mod

```diff
@@ -108,7 +108,7 @@ replace (
 	k8s.io/kube-scheduler => github.com/kubeedge/kubernetes/staging/src/k8s.io/kube-scheduler v1.22.6-kubeedge3
 	k8s.io/kubectl => github.com/kubeedge/kubernetes/staging/src/k8s.io/kubectl v1.22.6-kubeedge3
 	k8s.io/kubelet => github.com/kubeedge/kubernetes/staging/src/k8s.io/kubelet v1.22.6-kubeedge3
-	k8s.io/kubernetes => github.com/kubeedge/kubernetes v1.22.6-kubeedge3
+	k8s.io/kubernetes => github.com/vincentgoat/ke-kubernetes v1.22.6-kubeedge-prober
```

Comment:
Maybe commit the change of `github.com/vincentgoat/ke-kubernetes@v1.22.6-kubeedge-prober` to `github.com/kubeedge/kubernetes`, and then fix this PR

Reply:
Yes, after this PR is reviewed.